src/tests/integration-test

```diff
@@ -1264,6 +1264,10 @@ class Smart(UDisksTestCase):
 
 # ----------------------------------------------------------------------------
 
+def _bytes_to_ay(bytes):
+    return [i for i in bytes]
```

Comment:
Would a generator work as well?

Reply:
No, because pygobject is doing this: `if not isinstance(value, (dict, tuple, list)):`